### PR TITLE
Fix get_scoring method (issue #59)

### PR DESCRIPTION
**Pull request by kxk302**
### Pull Request Checklist

- [X ] Check python style, using `flake8 ./galaxy_ml`.
- [ X] Run API tests, using `cd galaxy_ml; nosetests ./tests -sv`.
- [ ] Check tool lint, using `cd galaxy_ml/tools; planemo lint`.
- [ ] Run tool tests, using `cd galaxy_ml/tools; planemo test`.
- [ ] Update `docs/CHANGELOG`, if applicable.
- [ ] Update `README.md`, if applicable.

### CODE REVIEW

**Comment by kxk302** [comment]
This PR is for this issue: https://github.com/goeckslab/Galaxy-ML/issues/59

File: galaxy_ml/utils.py [hunk 1]
```diff
@@ -602,6 +602,12 @@ def get_scoring(scoring_json):
     if scoring_json.get('secondary_scoring', None) not in (None, 'None', '')\
             and scoring_json['secondary_scoring'] !=\
             scoring_json['primary_scoring']:
+        # secondary_scoring used to be a comma separated string.
+        # At some point it turned into a list. Adding logic here
+        # to support both list and string (for backward compatability)
+        if type(scoring_json['secondary_scoring']) is list:
+            scoring_json['secondary_scoring'] =\
+                ",".join(scoring_json['secondary_scoring'])
         scoring = set(scoring_json['secondary_scoring'].split(','))
```
Comment (qiagu) [inline galaxy_ml/utils.py:611]: @kxk302  Thanks for making this.
How about this way?
```suggestion
        scoring = scoring_json['secondary_scoring']
        if not isinstance(scoring, list):
            scoring = scoring.split(',')
        scoring = set(scoring)
```

Comment (kxk302) [reply galaxy_ml/utils.py:611]: This is good too.

**Comment by qiagu** [comment]
@kxk302 Thanks

**Comment by kxk302** [comment]
Thanks @qiagu!

**Comment by kxk302** [comment]
@qiagu No rush, but whenever you deploy this change, could you please let me know, so I can update the galaxytools? Thanks.

**Comment by qiagu** [comment]
@kxk302  Sure. If no special needs, this change will be come with the next major upgrade, v0.9.x. Does it sound good to you?

**Comment by kxk302** [comment]
> @kxk302 Sure. If no special needs, this change will be come with the next major upgrade, v0.9.x. Does it sound good to you?

Yes, that is perfectly fine. I have a fix in galaxytools already. When this change is released, I can take out that temporary fix. 
Thank you!

**Pull request merged**